app/utils/json_handler.py

```
@@ -0,0 +1,154 @@
+                json.dump([], f)
+    
+    def read(self) -> list[dict[str, Any]]:
+        """
+        JSON 파일에서 데이터 읽기
+        
+        Returns(반환값):
+            list[dict[str, Any]]: 데이터 리스트
+        """
+        lock = FileLock(self.lock_path, timeout=10)
+        with lock:
+            with open(self.file_path, 'r', encoding='utf-8') as f:
```

Ruff (0.14.11)

Unnecessary mode argument

Remove mode argument

(UP015)